drivers/power/pm/Kconfig

```diff
@@ -29,6 +29,12 @@ config PM_PROCFS
 	---help---
 		Enable procfs for pm.
 
+config PM_RUNTIME
+	bool "PM runtime support"
+	default n
+	---help---
+		Enable pm runtime
```

Comment:
Please include an informative help here, describe the purpose of the PM Runtime

Reply:
done

Reply:
@acassis  Please view again